src/components/SingleRev.jsx

```diff
@@ -102,40 +110,57 @@ export const SingleRev = () => {
   const handlePostButton = (text) => {
     setLoadingComm(true);
     const regex = /^(?!\s)[\s\S]*$/;
-    if (!regex.test(text.body)) {
+    if (!regex.test(text.body) || text.body === ``) {
       alert("Please start typing a text");
       setLoadingComm(false);
+      setPosting(``);
     } else {
       const sentComment = { id: text.id, username: username, body: text.body };
       postComment(sentComment).then((res) => {
         setComments((currComm) => {
           setLoadingComm(false);
-          alert("Comment posted successfully");
+          setPosting(``);
           return [res, ...currComm];
         });
       });
     }
   };
 
   const handleDeleteComment = (id) => {
-    setDeletingComm(true)
-    setLoadingComm(true)
+    setDeleteMsg(true);
+    setDeletingComm(true);
+    setLoadingComm(true);
     deleteComment(id)
       .then((res) => {
         setDeletingComm(false);
-        setLoadingComm(false)
-        alert(`Comment deleted`)
+        setLoadingComm(false);
+        setPosting(``);
+        setTimeout((func)=>{
+          setDeleteMsg(false)
+        }, 3000);
       })
       .catch((err) => {
         alert(`Could not delete comment`);
         setDeletingComm(false);
-    setLoadingComm(false)
+        setLoadingComm(false);
+        setPosting(``);
+        setDeleteMsg(false);
       });
   };
-console.log(`loading`,loadingComm);
-console.log(`del`,deletingComm);
+  const date = new Date(oneRev.created_at);
```

Comment:
There are more efficient ways of parsing a date object, but admirable effort here. I'd advise looking through the docs for Data, or even string methods for manipulating it that way